associacao-brasileira-de-normas-tecnicas-ufes-initials.csl

```diff
@@ -2,26 +2,56 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="pt-BR" demote-non-dropping-particle="never">
   <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Universidade Federal do Espírito Santo - ABNT (Portuguese - Brazil)</title>
-    <title-short>UFES-ABNT-Autoria_Abreviada</title-short>
-    <id>http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufes</id>
-    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufes" rel="self"/>
-    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufmg-face-initials" rel="template"/>
-    <!-- Este modelo UFES-ABNT foi realizado por meio da modificação do FACE/UFMG -->
+    <title>Universidade Federal do Espírito Santo - ABNT (autoria abreviada) (Portuguese - Brazil)</title>
+    <title-short>ABNT-UFES</title-short>
+    <id>http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufes-initials</id>
+    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufes-initials" rel="self"/>
+    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas" rel="template"/>
```

Comment:
@alosaulo, didn't you base this style off of http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufmg-face-initials instead of http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas? The XML comment below reads "Este estilo UFES-ABNT foi realizado principalmente tendo como base o estilo  FACE/UFMG"

Reply:
@alosaulo could you check on this question by rmzelle, please? I think this is mostly ready otherwise.